.github/workflows/ghcr.yaml

```diff
@@ -58,6 +59,22 @@ jobs:
           tags: ${{ steps.meta-watcher.outputs.tags }}
           labels: ${{ steps.meta-watcher.outputs.labels }}
 
+      - name: Extract metadata (tags, labels) for Docker (Webhook)
```

Comment:
i think there is.a way to do "matrix" in google actions for not have to copy and paste large block everytime, i guess we can do that if we have to  add another image again 🙃

Reply:
we can use matrix but there will 4 jobs created currently we execute all in one